odonto/odonto_submissions/serializers.py

```diff
@@ -207,6 +208,16 @@ def __init__(self, model_instance):
         "Patient declined": t.ETHNIC_ORIGIN_PATIENT_DECLINED
     }
 
+    def forename(self):
+        return clean_non_alphanumeric(
+            self.model_instance.first_name
+        ).upper()
```

Comment:
Why doesn't the `clean_` function do the uppercasing ?
It mentions uppercase only in the docstring then doesn't do anything about it :/
The address code below also implements `clean_` inline then chains `.upper()` in a forloop BTW..

Reply:
Maybe b/c apparently names have no spaces ?

Reply:
So yeah Address is a different case as it allows for internal spaces. The question of whether the clean method should upper case is a great one. The reason it doesn't is because I couldn't think of a good name for the method if it did the upper casing as well 😞 

however if that's the case the doc string definitely shouldn't mention upper casing.

<img width="580" alt="Screen Shot 2019-08-09 at 12 02 49" src="https://user-images.githubusercontent.com/2175455/62774747-a5743900-ba9d-11e9-81b7-76d962e62138.png">